kie-based-services/src/main/java/io/elimu/genericapi/service/MixPanelEventListener.java

```diff
@@ -62,7 +62,21 @@ protected void identify(ServiceRequest serviceRequest) {
 			this.otherData.clear();
 			for (String headerName : serviceRequest.getHeaders().keySet()) {
 				if (headerName.startsWith("mixpanel-")) {
-					this.otherData.put(headerName.replace("mixpanel-", ""), serviceRequest.getHeader(headerName));
+					String startPoint = headerName.replace("mixpanel-", "");
```

Comment:
@marianbuenosayres looks like this is duplication of code. Can we move prettify function in common utility class and reuse at all places